crates/ln-dlc-node/src/node/invoice.rs

```diff
@@ -175,7 +176,8 @@ where
             }
             Err(PaymentError::Sending(err)) => {
                 tracing::error!(?err, "Failed to send payment");
-                HTLCStatus::Failed
+                let failure_reason = retryable_send_failure_to_string(err);
+                anyhow::bail!("Failed to send payment: {failure_reason}");
```

Comment:
🔧 I think we still want to store the failed payment.

Reply:
do we really? the user got the info straight away that the payment failed

I'm happy to add it, but I'm not sure whether it's useful.

Reply:
I'll store it now to preserve the old behaviour, so it's less surprising.